bili.config.js

```diff
@@ -25,6 +25,13 @@ module.exports = {
     alias: {
       resolve: ['.vue', '.js'],
       entries: [{ find: '@', replacement: path.join(__dirname, './src') }]
+    },
+    copy: {
+      targets: [
+        { src: 'src/server/index.js', dest: 'dist/server' },
+        { src: 'src/server/apiVideo/**', dest: 'dist/server/apiVideo' },
```

Comment:
Can we achieve this with:
```suggestion
        { src: 'src/server', dest: 'dist/server' },
```
?

Reply:
`{ src: 'src/server', dest: 'dist' },`
This works